code/game/machinery/shieldgen.dm

```diff
@@ -419,8 +419,8 @@
 		needs_power = TRUE
 		setDir(get_dir(gen_primary, gen_secondary))
 	for(var/mob/living/L in get_turf(src))
-		visible_message("<span class='danger'>\The [src] is suddenly occupying the same space as \the [L]!</span>")
-		L.gib()
+		visible_message("<span class='danger'>\The [src] zaps through \The [L]!</span>")
+		L.electrocute_act(30, src, 1, FALSE, FALSE, FALSE, FALSE, TRUE)
```

Comment:
```suggestion
		L.electrocute_act(30, src)
```
Litterally all the other arguments you put in there are their default values anyways except the first two obciously.

Reply:
also i just realized there was a FALSE to many in there anyways so the TRUE would have not matched up with the stun argument

Reply:
or not i am confused one second